### ISSUE TITLE
It’s down again. 

### ISSUE THREAD

**Issue by tetra59** [p0ody/ff2ebook#92]
It’s not downloading fics, old or new. Not even reading from archives. It won’t get info.

**Comment by p0ody** [comment]
Cloadflare changed again (a small change this time at least)...  It should be back up and working now

**Comment by tetra59** [comment]
Yes! You’re the best. I’d noticed cloudflare was really working overtime but I assumed it was because it didn’t like my Adblocker, which I refuse to disable. Thx.

**Comment by tetra59** [comment]
Yes! You’re the best. I’d noticed cloudflare was really working overtime but I assumed it was because it didn’t like my Adblocker, which I refuse to disable. Thx.

**Comment by StarWolf3000** [comment]
Cloudflare seems to be up with something again, fetching data doesn't work.

**Comment by p0ody** [comment]
Somehow docker crashed an did not restart itself, should be fine now
